### ISSUE TITLE
i386 & armhf releases

### ISSUE THREAD

**Issue by moondeck** [Phoneboard/Phoneboard#14]
Are i386 and armhf/aarch64 releases possible without breaking stuff? This could be useful for people using old computers as workshop machines, and people looking into using something like the Raspberry Pi (or other *Pi  boards).

I am willing to test it on all three architectures should it be needed.

**Comment by guidopola** [comment]
@moondeck If that arch + OS have QT support it should not be a problem to port it.
For example the windows version it's compiled with mingw32 so it's i386 ;)

**Comment by moondeck** [comment]
Nice, armhf/arm64 does have QT support, so does i386. Could you try to compile for armhf for the next release?

BTW, I've shot you an email.

**Comment by guidopola** [comment]
Hey @moondeck do you still interested in this port?, I'm currently without internet connection but I might give it a shot this weekend if you want to

**Comment by moondeck** [comment]
Yes, very much. If you can, you could also try building for aarch64, as the newer xPi boards are that.
